src/publish-beta/compile.ts

```diff
@@ -0,0 +1,14 @@
+// publish-beta/compile.ts
+
+import childProcess from 'node:child_process';
+import util from 'node:util';
+import debug from 'debug';
+
+const log = debug('action:compile');
```

Comment:
I think all the debug namespaces should be prefixed by the name of the action, e.g. `publish-beta:compile`